### PR TITLE
Rewrite image inlining as an AsciiDoc extension

### PR DESCRIPTION
**Pull request by lexaknyazev**

### CODE REVIEW

**Comment by MarkCallow** [comment]
@lexaknyazev , you need to fix the build.

File: Makefile [hunk 1]
```diff
@@ -3,29 +3,19 @@
...
 
 $(out)/ktx.pdf:
 
-$(intermediate) $(out):
-	mkdir -p $@
-
 clean:
-	rm -rf $(intermediate)
-
-clobber: clean
```
Comment (MarkCallow) [inline Makefile]: Still need both clean: & clobber: targets. clean will do nothing. clobber should rm $(out).

**Comment by lexaknyazev** [comment]
Fixed `Makefile`. Travis is working now.

File: inline-images.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    process do |document, output|
+      _replace_urls(output) if document.basebackend? 'html'
+    end
+  end
+end
+
+MIME_TYPES = {
+    '.png' => 'image/png',
+    '.svg' => 'image/svg+xml',
+    '.ico' => 'image/x-icon'}.freeze
```
Comment (MarkCallow) [inline inline-images.rb]: Nit. Please move closing `}.freeze` to a new line. It is too easy to miss the closing now.

**Pull request merged**